BMM.UI.Android/Application/Bindings/MvxCardCircle.cs

```diff
@@ -22,7 +24,11 @@ protected override void SetValueImpl(object target, object value)
         {
             var card = (CardView)target;
             if (value is bool test && test)
-                card.Radius = 10000; // this takes pixel whereas most other units are in dp
+            {
+                int imageHeightInDp = 160; // should be the same value as in listitem_tracklist_header.xml
```

Comment:
I think you could get the defined height from the card view like this `card.LayoutParameters.Width` which would already be in pixels

Reply:
Wow. That's so much better.
@olivereisenhut thanks